lib/assessment/rubric_tester.py

```diff
@@ -302,57 +306,54 @@ def main():
-        params = get_params(experiment_lesson_prefix)
+        params = get_params(params_lesson_prefix)
         response_type = params.get('response-type', 'tsv')
-        prompt, standard_rubric = read_inputs(prompt_file, standard_rubric_file, experiment_lesson_prefix)
+        prompt, standard_rubric = read_inputs(prompt_file, standard_rubric_file, params_lesson_prefix)
         student_files = get_student_files(options.max_num_students, dataset_lesson_prefix, student_ids=options.student_ids)
         if os.path.exists(os.path.join(dataset_lesson_prefix, actual_labels_file_old)):
             actual_labels = get_actual_labels(actual_labels_file_old, dataset_lesson_prefix)
         else:
             actual_labels = get_actual_labels(actual_labels_file, dataset_lesson_prefix)
-        examples = get_examples(experiment_lesson_prefix, response_type)
+        examples = get_examples(params_lesson_prefix, response_type)
 
         validate_rubrics(actual_labels, standard_rubric)
         validate_students(student_files, actual_labels)
         rubric = standard_rubric
 
         # set up output and cache directories
-        os.makedirs(os.path.join(experiment_lesson_prefix, output_dir_name), exist_ok=True)
-        os.makedirs(os.path.join(experiment_lesson_prefix, cache_dir_name), exist_ok=True)
+        os.makedirs(os.path.join(params_lesson_prefix, output_dir_name), exist_ok=True)
+        os.makedirs(os.path.join(params_lesson_prefix, cache_dir_name), exist_ok=True)
         if not options.use_cached:
-            for file in glob.glob(f'{os.path.join(experiment_lesson_prefix, cache_dir_name)}/*'):
+            for file in glob.glob(f'{os.path.join(params_lesson_prefix, cache_dir_name)}/*'):
                 os.remove(file)
 
         # call label function to either call openAI or read from cache
         with concurrent.futures.ThreadPoolExecutor(max_workers=options.workers) as executor:
-            predicted_labels = list(executor.map(lambda student_file: read_and_label_student_work(prompt, rubric, student_file, examples, options, params, experiment_lesson_prefix, response_type), student_files))
+            predicted_labels = list(executor.map(lambda student_file: read_and_label_student_work(prompt, rubric, student_file, examples, options, params, params_lesson_prefix, response_type), student_files))
 
         errors = [student_id for student_id, labels in predicted_labels if not labels]
         # predicted_labels contains metadata and data (labels), we care about the data key
         predicted_labels = {student_id: labels['data'] for student_id, labels in predicted_labels if labels}
 
         for is_pass_fail in [True, False]:
             output_filename = 'report-pass-fail.html' if is_pass_fail else 'report-exact-match.html'
-            output_file = os.path.join(experiment_lesson_prefix, output_dir_name, output_filename)
+            output_file = os.path.join(params_lesson_prefix, output_dir_name, output_filename)
```

Comment:
I'm concerned about omitting the experiment name from the report here (and other places). this seems like an essential piece of information to include in the report. one idea would be to start using the branch name that the local  aitt_release_data repo is currently on. this can be obtained via something like `cd ../aitt_release_data; git branch` or by parsing the branch name from this file in that repo:
```
Dave-MBP:~/src/aiproxy (main)$ cat .git/HEAD
ref: refs/heads/main
```
I'd go so far to say that if the current branch is "main" then we should raise an error. in the future we could consider allowing you to be on the main branch if there is a release tag for the current commit, and then using that as the experiment name.

There is the argument that the experiment name is not needed if all the parameters it contains are included in the report, but this seems like it could be a bit annoying because it could lead us to have to hunt for an experiment with the parameters shown in the report.

I think the bottom line is that the report should contain enough information to be repeatable, but I'm open to pushback here on what that looks like.

Reply:
Good points. I'll add this